src/_rules/shadow.js

```diff
@@ -9,4 +11,17 @@ const boxShadowDefaults = {
 
 export const shadows = [
   [/^shadow-(small|medium|large|xlarge)$/, ([, size]) => ({ 'box-shadow': `var(--w-shadow-${size}, ${boxShadowDefaults[size]})` })],
-];
\ No newline at end of file
+];
+
+const dropShadowDefaultColor = '64, 64, 64';
+
+const dropShadowDefaults = {
+  small: [`drop-shadow(rgba(${dropShadowDefaultColor}, .16) 0 1px 6px) drop-shadow(rgba(${dropShadowDefaultColor}, .24) 0 1px 1px)`],
+  medium: [`drop-shadow(rgba(${dropShadowDefaultColor}, .24) 0 3px 8px) drop-shadow(rgba(${dropShadowDefaultColor}, .16) 0 3px 6px)`],
+  large: [`drop-shadow(rgba(${dropShadowDefaultColor}, .23) 0 6px 8px) drop-shadow(rgba(${dropShadowDefaultColor}, .19) 0 10px 20px)`],
+  xlarge: [`drop-shadow(rgba(${dropShadowDefaultColor}, .22) 0 9px 12px) drop-shadow(rgba(${dropShadowDefaultColor}, .25) 0 14px 28px)`],
```

Comment:
Should we maybe rename these to -s, -m, -l and -xl as discussed at standup? Would be great if you could also add a todo right next to box-shadow rules to do the same when we have time (or add a task wherever it makes sense). Wdyt?